packages/editor-ui/src/components/CredentialCard.vue

```diff
@@ -62,7 +64,10 @@ const actions = computed(() => {
 		});
 	}
 
-	if (credentialPermissions.value.move) {
+	if (
+		credentialPermissions.value.move &&
+		String(settingsStore.planName).toLowerCase() !== 'community'
```

Comment:
Would be worth adding `isCommunityPlan` in the settings. We're likely to reuse this.

Reply:
Added the new getter to settings store